Makefile

```diff
@@ -1,5 +1,5 @@
-start: ## Builds and starts the project in a docker container
-	@docker-compose up --remove-orphans --build
+install-node-modules: ## Install dependencies locally
```

Comment:
Can we name this just `install`?

Reply:
We can, I just wanted to point out that it is not doing anything on the backend.